app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict[str, any]]) -> list[Person]:
+    person_dict = {person["name"]: Person(person["name"],
+                                          person["age"]) for person in people}
```

Comment:
```suggestion
    person_dict = {
        person["name"]: Person(person["name"], person["age"]) 
        for person in people
    }
```
I would prefer this style